src/scss/components/_gameSection.scss

```diff
@@ -7,6 +7,19 @@
   border-bottom-right-radius: 20px;
   margin: 0 40px;
   padding: 35px 40px;
+
+  &__questionAndTimer {
+    width: 100%;
+    display: flex;
+    justify-content: flex-end;
+    align-items: center;
+
+    & > *:first-child {
+      margin: 0 auto;
+      padding-left: 90px;
+      justify-self: center;
+    }
```

Comment:
Tak zostać nie może :))

Reply:
To napisał byś dlaczego nie może xD

Reply:
Konieczny jest tu operator '*'?

Reply:
Sprawdzę to ale chyba masz rację i nie jest konieczny :D